src/olympia/addons/tests/test_views.py

```diff
@@ -1733,6 +1784,84 @@ def test_delete_icon(self, remove_icons_mock):
         assert alog.user == self.user
         assert alog.action == amo.LOG.CHANGE_MEDIA.id
 
+    def _test_metadata_content_review(self):
+        response = self.client.patch(
+            self.url,
+            data={'name': {'en-US': 'new name'}, 'summary': {'en-US': 'new summary'}},
+        )
+        assert response.status_code == 200
+
+    @override_switch('metadata-content-review', active=False)
+    @mock.patch('olympia.addons.serializers.fetch_translations_from_addon')
+    def test_metadata_content_review_waffle_off(self, fetch_mock):
+        self._test_metadata_content_review()
+
+        fetch_mock.assert_not_called()
+        with self.assertRaises(AssertionError):
```

Comment:
Just a comment: This seems like an ugly way to assert "not called", but I assume a simpler version does not exist or we'd use it.

Reply:
yep.  You can `assert_not_called` on a mock, but asserting a _specific_ call wasn't made, while other calls were, means either doing this (ugly but at least clear, imo) or iterating through all the calls, extracting the call kwargs and args, and asserting that none of the calls have this arg.